_works/redbullmo44.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+category: work
+title: 'Red Bull MO44'
+subtitle: 'Website'
+client: 'Red Bull'
+tags: 'website'
+image:
+ - url: '../../public/images/work/thumbs/thumb-redbullMO44.jpg'
+links:
+ - type: website
+   link: http://mo44.redbull.pt/
+sitemap: false
+---
+
+We´ve built a website were helped you to win official merchandising of Red Bull KTM.
```

Comment:
We've helped Red Bull Portugal build a racing game within the context of the MotoGP2.

You can race as Miguel Oliveira (a Portuguese and Red Bull sponsored MotoGP2 racer) and win prizes that go from merchandising to an all expenses paid trip to see a live race.